backend/api/swagger/pipeline.swagger.json

```diff
@@ -322,7 +335,7 @@
         ]
       },
       "delete": {
-        "summary": "Delete a pipeline.",
+        "summary": "Delete a pipeline. All pipeline versions under this pipeline will be deleted as well.",
```

Comment:
Suggested revision:

Deletes a pipeline and its pipeline versions.

Reply:
addressed in proto file